.config/mise/config.toml

```diff
@@ -0,0 +1,3 @@
+[tools]
+java = '24'
+node = '22.14.0'
```

Comment:
Should we add python (3.13) as well?

Reply:
Yes, but not yet. It's not required to run the app, so I left it out for now. Will add it in a follow-up